src/common.js

```diff
@@ -55,18 +55,24 @@ async function loadNimiq() {
     }));
 }
 
+/**
+ * @typedef {{loadNimiq: boolean, whitelist: string[]}} Options
+ */
+
 /**
  * @template {KeyguardRequest.RedirectRequest} T
  * @param {Newable} RequestApiClass - Class object of the API which is to be exposed via RPC
- * @param {object} [options]
+ * @param {Partial<Options>} [opts]
```

Comment:
Because they do exist on the `Options` type and would otherwise also be `?` when used in the rest of the function and would require type annotations to force that they are indeed defined.